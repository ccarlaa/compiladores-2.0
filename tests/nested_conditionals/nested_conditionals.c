int main() {
    int a = 10;
    int b = 5;

    if (a > b) {
        printf("a is greater than b");
        if (a > 15) {
            printf(" and a is greater than 15");
        } else {
            printf(" and a is not greater than 15");
        }
    } else if (b > a) {
        printf("b is greater than a");
    } else {
        printf("a and b are equal");
    }
    return 0;
}