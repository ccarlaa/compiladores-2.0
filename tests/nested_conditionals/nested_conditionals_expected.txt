programa
{
    funcao inicio()
    {
        inteiro a = 10
        inteiro b = 5

        se (a > b)
        {
            escreva("a is greater than b")
            se (a > 15)
            {
                escreva(" and a is greater than 15")
            }
            senao
            {
                escreva(" and a is not greater than 15")
            }
        }
        senao se (b > a)
        {
            escreva("b is greater than a")
        }
        senao
        {
            escreva("a and b are equal")
        }
        // retorne 0
    }
}